src/internal/m365/collection/drive/url_cache.go

```diff
@@ -19,7 +19,9 @@ import (
 
 const (
 	urlCacheDriveItemThreshold = 300 * 1000
-	urlCacheRefreshInterval    = 1 * time.Hour
+	// 600 pages = 300k items, since delta enumeration produces 500 items per page
```

Comment:
is it 500 or 999? I thought we were going to bump it back up?

Reply:
iirc it's 500 max for deltas, 1k max for non-deltas.  I'll have to go look at microsoft documentation again, though.

Reply:
Confirming what Keepers said. 

Btw, we have `maxDeltaPageSize` and `maxNonDeltaPageSize` defined in api pkg. We should export them and use here instead of hardcoding 600. This will protect against future changes to delta page size.

Reply:
oh wild. Makes me wish they had consistent limits across APIs 😂

Reply:
The limits are consistent actually, lol. We are just using different limits, tbh I don't have a good reason why they have to be different( 500 vs 999). We do have a pending ticket to figure out the best page size.